### ISSUE TITLE
NTR, pulmo: Local pulmonary hemorrhage 

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5622]
**Preferred term label:**
Localized pulmonary hemorrhage 

**Synonyms**

**Definition (free text, please give PubMed ID)**
Circumscribed pulmonary hemorrhage originating from a single bleeding site in the lungs. This can be due to infections, tumorigenesis, foreign bodies, or vascular abnormalities. Patient often feel the site of bleeding, contrast CT scan or angiography may localize the bleeder. 

**Parent term (use hpo.jax.org/app)**
Parent Term: Pulmonary hemorrhage HP:0040223. 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
LMU:chrapp, @katrinknoflach, @mgriese

**Comment by pnrobinson** [comment]
added as HP:0032990
